src/api/lib/crypto/crypto.rb

```diff
@@ -0,0 +1,34 @@
+module Crypto
+  module Encrypt
+    def self.cloud_upload_data(data)
+      crypto = GPGME::Crypto.new(armor: true)
+      crypto.encrypt(data, recipients: PublicKeys.cloud_upload.fingerprint, always_trust: true).to_s
+    end
+  end
+
+  module PublicKeys
```

Comment:
Maybe it would make sense to move this to the AzureConfiguration?
There is only one Public/Secret key pair, right?

Reply:
Well it's called `cloud upload public key` (also according to the backend route: `/cloudupload/_pubkey`) and as we already talked about encrypting the EC2 stuff as well, why not keeping it generic?

Reply:
We won't encrypt the EC2 configuration, I don't remember that we talked about that :thinking: 

If it is only related to cloud, it shouldn't be in a lib! Then create a model e.g. ``Cloud::Backend::PublicKey``. And then you can e.g. do ``Cloud::Backend::PublicKey.get``. 

And I would like to get rid of the cache and store it in a variable. There should be a way that also Rubocop won't complain!